packages/lodestar/src/network/gossip/gossipsub.ts

```diff
@@ -94,6 +94,8 @@ export class Eth2Gossipsub extends Gossipsub {
 
     const scoreParams = computeGossipPeerScoreParams(modules);
 
+    const heartbeatInterval = 0.7 * 1000;
```

Comment:
This should be a constant at the top in all caps, with a link to the spec page that defines it